hackathon/models.py

```diff
@@ -42,6 +42,8 @@ class Hackathon(models.Model):
                                   blank=True,
                                   on_delete=models.SET_NULL,
                                   related_name="hackathon_organiser")
+    # Status field which can be set to deleted to filter out deleted hackathons from frontend list
+    status = models.CharField(max_length=50, blank=True, null=True)
```

Comment:
Also, this should use a list of choices. You can have a look at @auxfuse's PR and just adjust to the same definition: https://github.com/Code-Institute-Community/ci-hackathon-app/pull/101

Reply:
Cool, so from the PR that you've tagged, it looks like a status field has already been added to the Hackathon model in that PR, with the relevant choices. With this being the case, I'll remove my change from this file so that it doesn't cause any merge conflicts. Then, once both PRs have been merged in to the master branch, it should work fine - the logic in the delete_hackathon view is already correct to work with the relevant status choices - sets the status to 'deleted', which is one of the choices. If that's cool with you, just let me know and I'll go ahead and remove my changes to this file and push it.

Reply:
Alternatively, I'm happy to wait until the other PR has been merged in. Once done, I can then pull that in and test that my code integrates correctly with it before pushing any of my final changes to this PR. I'm cool with either option, just let me know please 👍

Reply:
@stefdworschak This is now integrated with the latest PR - tested and working.